gwt/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1610,6 +1641,53 @@ private void setConnectionSpecificInfo( IDatabaseConnection meta ) {
         webAppName.setValue( meta.getDatabaseName() );
       }
     }
+
+    if ( jdbcAuthMethod != null ) {
+      jdbcAuthMethod.setValue( "breakpoint here" );
```

Comment:
^

Reply:
just making sure **you** were paying attention